### DISCUSSION TITLE
Open Source Offline Desktop App

### DISCUSSION THREAD

**Discussion by sirinath** [typst/typst#196] [Ideas]
It would be a great addition so that Typst can be completely used locally offline without having a Typst account. 

And for those times ones want to work online the ability to connect with the Typst account and have the same experience as working online.

Ability to mix and match the above usage for different documents and projects.

**Comment by miraclemoses** [comment]
Great innovation

**Comment by ichxorya** [comment]
I think it could be a simple Tauri project with the web interface embeded. It should also be extended to support local editing too.

**Reply by Cubxity** [reply]
I'm currently working on one - https://github.com/Cubxity/typstudio. However, it doesn't use the web interface.

**Comment by kamasylvia** [comment]
I think VSCode with typst plug-in is enough ^_^
